front/app/modules/commercial/content_builder/admin/components/ContentBuilderSettings/index.tsx

```diff
@@ -42,10 +42,14 @@ const ContentBuilderSettings = () => {
     };
   });
 
-  const getComponentNameMessage = (name: 'Container') => {
+  const getComponentNameMessage = (name: 'Container' | 'Text') => {
     switch (name) {
       case 'Container':
         return messages.oneColumn;
+      case 'Text':
+        return messages.text;
+      default:
+        return messages.oneColumn;
```

Comment:
I think this "default" issue is solved in your two column work, so I'll change this once I do the merging.